web/src/components/DatePicker.tsx

```diff
@@ -74,6 +74,7 @@ export type CustomDatePickerProps = {
   date?: DateTime | null
   setDate: (date: DateTime | null) => void
   error?: string
+  placeholderDate?: DateTime
```

Comment:
🔧 Since we always pass a placeholder date (and it is best practice to do so), we should make this required.
```suggestion
  placeholderDate: DateTime
```